.pre-commit-config.yaml

```diff
@@ -26,3 +26,12 @@ repos:
     hooks:
       - id: flake8
         additional_dependencies: [flake8-isort]
+
```

Comment:
what does this do?

Reply:
tbh I am not sure what this adds, it could be it adds additional functionality that isort doesn't have OR it could be used instead, for now though I will remove it as I am not sure what its doing (I added it as its part of our work commit hooks, but not sure if thats just a historical quirk or intentional)